vcd_cli/role.py

```diff
@@ -51,19 +54,48 @@ def list_roles(ctx):
               '--org',
               'org_name',
               required=False,
-              metavar='<org-name>',
+              metavar='<org_name>',
               help='name of the org')
-def list_rights(ctx, role_name, org_name=None):
+def list_rights(ctx, role_name, org_name):
```

Comment:
If org_name is optional param, we should provide a default value in the click.option argument list viz. None

Reply:
Yes, I did provide first as you can see, but I noticed that none of the other commands in vcd-cli have provided None value, so removed it to keep it consistent.